shared/src/main/scala/tastyquery/reader/classfiles/Classpaths.scala

```diff
@@ -0,0 +1,54 @@
+package tastyquery.reader.classfiles
+
+import tastyquery.ast.Names.{SimpleName, TermName, EmptyPackageName, termName}
+
+object Classpaths {
+
+  transparent inline def universe(using universe: Universe): Universe = universe
```

Comment:
Why is this `transparent`? Can't you achieve the same thing with
```suggestion
  def universe(using universe: Universe): universe.type = universe
```
?

Reply:
the choice for transparent inline is that you can use `universe` in an import prefix